README.md

```diff
@@ -0,0 +1,58 @@
+# Gomoss
+Go client for [Moss](http://theory.stanford.edu/%7Eaiken/moss/) plagiarism software detector.
+
+## Introduction
+This is Go interface for Moss client. It is written for [omegaUp](https://omegaup.com/) project
+
+## Installation
+`go get -u github.com/SpaceWhite/gomoss`
+
+## Usage
+```
+import "github.com/SpaceWhite/gomoss"
+import "encoding/json"
+
+func main() {
+    userID := 123456789
+    request := gomoss.NewRequest(userID)
+
+    // Add code to MOSS
+    for _, filename := range []string{"a.c", "b.c"} {
+        f, err := os.Open(path.Join("codes", filename))
+        if err != nil {
+            panic(err)
+        }
+        defer f.Close()
+        request.Code.Add(f, filename)
+    }
+
+    // Add base code to MOSS
+    sampleFile, _ := os.Open("codes/sample.c")
+    request.BaseCode.Add(sampleFile, "sample.c")
+    defer sampleFile.Close()
+
+    ctx := context.Background()
+
+    // Request to MOSS
+    url, err := m.Submit(ctx)
+    if err != nil {
+        panic(err)
+    }
+
+    // This will download all MOSS HTML to the temp directory
+    // downloadDir, err := gomoss.Download(url)
+
+    // This will download all MOSS results into array of map
+    // You don't necessarily need to run Download in order to run Extract
+    results := gomoss.Extract(url)
```

Comment:
nit: update the README.md to reflect the latest code. Or even better, so that I don't have to nit you every time to update this, can you move this to a [Go example](https://blog.golang.org/examples) test? That way, it always needs to be compilable, and it appears in godoc.